impl/src/main/java/com/okta/jwt/impl/jjwt/BaseVerifierBuilderSupport.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2018-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.jwt.impl.jjwt;
+
+import com.okta.commons.configcheck.ConfigurationValidator;
+import com.okta.jwt.VerifierBuilderSupport;
+import com.okta.jwt.impl.http.OkHttpClient;
+import io.jsonwebtoken.SigningKeyResolver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+abstract class BaseVerifierBuilderSupport<B extends VerifierBuilderSupport, R> implements VerifierBuilderSupport<B, R> {
+
+    private String issuer;
+    private long leeway = 120L;
+    private long connectionTimeout = 1000L;
+    private long readTimeout = 1000L;
+
+    String getIssuer() {
+        return issuer;
+    }
+
+    public B setIssuer(String issuer) {
+
+        // delay validation until 'validate' method is called
+        if (issuer != null) {
+            // trim and remove any trailing slash
+            this.issuer = issuer.trim().replaceAll("/$", "");
+        }
+        return self();
+    }
+
+    long getLeeway() {
+        return leeway;
+    }
+
+    public B setLeeway(long leeway) { // TODO use duration
```

Comment:
is this TODO up-to-date?

Reply:
Yes, but good call I didn't see this one when doing my last walk through